### DISCUSSION TITLE
How can i get the best results from it?

### DISCUSSION THREAD

**Discussion by iminahiljaved** [ashleykleynhans/runpod-worker-comfyui#4] [General]
I was using sdxl api before , the results i got from it were much better then in this, but due to some reasons i decided to shift. Now the output images are either not of good quality of deformed. can you suggest me what we can adjust?

**Comment by ashleykleynhans** [comment]
Not my job to tell you this, I don't even use ComfyUI, please don't waste my time with stupid questions.
